app/controllers/status_controller.rb

```diff
@@ -15,12 +15,23 @@ def index
   # This is useful for troubleshooting authentication with access tokens and
   # configuring proxies or load balancers.
   def whoami
-    render json: { 
+    audit_success
+
+    render json: {
       client_ip: request.ip,
       user_agent: request.user_agent,
       account: token_user.account,
       username: token_user.login,
       token_issued_at: Time.at(token_user.token.claims["iat"])
     }
   end
+
+  def audit_success
+    Audit.logger.log(
+      Audit::Event::Whoami.new(
+        client_ip: token_user.remote_ip,
+        role_id: ::Role.by_login(token_user.login, account: token_user.account),
```

Comment:
does `Role.by_login` return a `role` or `role_id`?

Reply:
right it's role ... I will rename it